[2025-10-22 14:11:30] [INFO] [django.utils.autoreload] Watching for file changes with StatReloader
[2025-10-22 14:12:51] [INFO] [django.server] "OPTIONS /api/auth/login/ HTTP/1.1" 200 0
[2025-10-22 14:12:52] [WARNING] [django.request] Bad Request: /api/auth/login/
[2025-10-22 14:12:52] [WARNING] [django.server] "POST /api/auth/login/ HTTP/1.1" 400 44
[2025-10-22 14:12:58] [INFO] [django.server] "POST /api/auth/login/ HTTP/1.1" 200 638
[2025-10-22 14:12:58] [INFO] [django.server] "OPTIONS /api/admin/dashboard/stats/ HTTP/1.1" 200 0
[2025-10-22 14:12:58] [INFO] [django.server] "OPTIONS /api/admin/dashboard/recent_activity/ HTTP/1.1" 200 0
[2025-10-22 14:12:58] [INFO] [django.server] "OPTIONS /api/admin/dashboard/recent_activity/ HTTP/1.1" 200 0
[2025-10-22 14:12:58] [INFO] [system] Dashboard stats requested by admin admin@safenetai.com
[2025-10-22 14:12:58] [INFO] [django.server] "GET /api/admin/dashboard/stats/ HTTP/1.1" 200 240
[2025-10-22 14:12:58] [INFO] [system] Dashboard stats requested by admin admin@safenetai.com
[2025-10-22 14:12:58] [INFO] [django.server] "GET /api/admin/dashboard/stats/ HTTP/1.1" 200 240
[2025-10-22 14:12:58] [INFO] [system] Recent activity requested by admin admin@safenetai.com
[2025-10-22 14:12:58] [INFO] [django.server] "GET /api/admin/dashboard/recent_activity/ HTTP/1.1" 200 8198
[2025-10-22 14:12:58] [INFO] [system] Recent activity requested by admin admin@safenetai.com
[2025-10-22 14:12:58] [INFO] [django.server] "GET /api/admin/dashboard/recent_activity/ HTTP/1.1" 200 8198
[2025-10-22 14:13:39] [INFO] [django.server] "POST /api/auth/login/ HTTP/1.1" 200 647
[2025-10-22 14:13:39] [INFO] [django.server] "OPTIONS /api/client/profile/me/ HTTP/1.1" 200 0
[2025-10-22 14:13:39] [INFO] [django.server] "OPTIONS /api/client/transactions/ HTTP/1.1" 200 0
[2025-10-22 14:13:39] [INFO] [django.server] "OPTIONS /api/client/transactions/ HTTP/1.1" 200 0
[2025-10-22 14:13:40] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:13:40] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 1754
[2025-10-22 14:13:40] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:13:40] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 1754
[2025-10-22 14:14:09] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 1754
[2025-10-22 14:14:09] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 1754
[2025-10-22 14:14:09] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:14:09] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:14:12] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:14:12] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 1754
[2025-10-22 14:14:32] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:14:32] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:14:39] [INFO] [system] System event: ML model loaded successfully | Component: fraud_ml_model | Severity: INFO | Data: {"model_path": "C:\\Users\\HP\\Desktop\\projects\\safeNetAi\\backend\\models\\fraud_isolation.joblib"}
[2025-10-22 14:14:39] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:14:39] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:14:39] [ERROR] [system] System event: Error making ML prediction | Component: fraud_ml_model | Severity: ERROR | Data: {"transaction_id": 324, "error": "X has 9 features, but StandardScaler is expecting 7 features as input."}
[2025-10-22 14:14:41] [INFO] [system] System event: Security OTP email sent successfully | Component: email_service | Severity: INFO | Data: {"user_email": "abassi.loay@gmail.com", "user_id": 71, "transaction_id": 324, "otp_code": "729472"}
[2025-10-22 14:14:41] [INFO] [system] System event: Transaction OTP created and sent successfully | Component: transactions | Severity: INFO | Data: {"user_email": "abassi.loay@gmail.com", "user_id": 71, "transaction_id": 324, "otp_id": 34}
[2025-10-22 14:14:41] [WARNING] [system] System event: Transaction requires OTP verification due to risk rules or AI assessment | Component: transactions | Severity: WARNING | Data: {"transaction_id": 324, "risk_score": 70, "triggers": ["\u26a0\ufe0f Effective distance violation: 101.17km > 60.0km threshold"], "ml_score": 0.5, "user_id": 71, "distance_violation": false, "otp_reasons": ["Business rules triggered: 1", "High risk score: 70", "Risk engine explicit requirement"], "reason": "risk_rules"}
[2025-10-22 14:14:41] [INFO] [django.server] "POST /api/client/transactions/ HTTP/1.1" 201 404
[2025-10-22 14:15:06] [INFO] [django.server] "OPTIONS /api/client/transactions/324/verify_otp/ HTTP/1.1" 200 0
[2025-10-22 14:15:06] [INFO] [system] System event: Transaction OTP verified successfully | Component: transactions | Severity: INFO | Data: {"user_email": "abassi.loay@gmail.com", "user_id": 71, "transaction_id": 324, "otp_id": 34}
[2025-10-22 14:15:08] [INFO] [system] System event: Transaction notification sent successfully | Component: email_service | Severity: INFO | Data: {"user_email": "abassi.loay@gmail.com", "user_id": 71, "transaction_id": 324, "status": "COMPLETED", "risk_level": "HIGH", "risk_score": 70}
[2025-10-22 14:15:08] [INFO] [django.server] "POST /api/client/transactions/324/verify_otp/ HTTP/1.1" 200 114
[2025-10-22 14:15:12] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:15:12] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:15:12] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 2049
[2025-10-22 14:15:12] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 2049
[2025-10-22 14:15:26] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:15:26] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:15:26] [INFO] [system] System event: ML model loaded successfully | Component: fraud_ml_model | Severity: INFO | Data: {"model_path": "C:\\Users\\HP\\Desktop\\projects\\safeNetAi\\backend\\models\\fraud_isolation.joblib"}
[2025-10-22 14:15:26] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:15:26] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:15:26] [ERROR] [system] System event: Error making ML prediction | Component: fraud_ml_model | Severity: ERROR | Data: {"transaction_id": 325, "error": "X has 9 features, but StandardScaler is expecting 7 features as input."}
[2025-10-22 14:15:28] [INFO] [system] System event: Transaction notification sent successfully | Component: email_service | Severity: INFO | Data: {"user_email": "abassi.loay@gmail.com", "user_id": 71, "transaction_id": 325, "status": "COMPLETED", "risk_level": "LOW", "risk_score": 20}
[2025-10-22 14:15:28] [INFO] [django.server] "POST /api/client/transactions/ HTTP/1.1" 201 107
[2025-10-22 14:16:13] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:16:13] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:16:13] [INFO] [system] System event: ML model loaded successfully | Component: fraud_ml_model | Severity: INFO | Data: {"model_path": "C:\\Users\\HP\\Desktop\\projects\\safeNetAi\\backend\\models\\fraud_isolation.joblib"}
[2025-10-22 14:16:13] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:16:13] [INFO] [system] System event: RiskEngine initialized | Component: risk_engine | Severity: INFO | Data: {"thresholds_count": 13, "rules_count": 4}
[2025-10-22 14:16:13] [ERROR] [system] System event: Error making ML prediction | Component: fraud_ml_model | Severity: ERROR | Data: {"transaction_id": 326, "error": "X has 9 features, but StandardScaler is expecting 7 features as input."}
[2025-10-22 14:16:14] [INFO] [system] System event: Security OTP email sent successfully | Component: email_service | Severity: INFO | Data: {"user_email": "abassi.loay@gmail.com", "user_id": 71, "transaction_id": 326, "otp_code": "624404"}
[2025-10-22 14:16:14] [INFO] [system] System event: Transaction OTP created and sent successfully | Component: transactions | Severity: INFO | Data: {"user_email": "abassi.loay@gmail.com", "user_id": 71, "transaction_id": 326, "otp_id": 35}
[2025-10-22 14:16:14] [WARNING] [system] System event: Transaction requires OTP verification due to risk rules or AI assessment | Component: transactions | Severity: WARNING | Data: {"transaction_id": 326, "risk_score": 65, "triggers": ["Large transfer: 12000.00 DZD > 5000.0 DZD", "Statistical outlier: z-score 2.43 > 2.0"], "ml_score": 0.5, "user_id": 71, "distance_violation": false, "otp_reasons": ["Business rules triggered: 2"], "reason": "risk_rules"}
[2025-10-22 14:16:14] [INFO] [django.server] "POST /api/client/transactions/ HTTP/1.1" 201 366
[2025-10-22 14:16:36] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:16:36] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 2639
[2025-10-22 14:16:36] [INFO] [django.server] "GET /api/client/profile/me/ HTTP/1.1" 200 467
[2025-10-22 14:16:36] [INFO] [django.server] "GET /api/client/transactions/ HTTP/1.1" 200 2639
